packages/paste-website/src/pages/patterns/delete/index.mdx

```diff
@@ -148,13 +164,50 @@ For high-severity deletions, show a confirmation modal that explains what is bei
   </Disclosure>
 </Box>
 
-## Post-deletion
-
-After the user has deleted the object, navigate them to the index page, where they can see a list of all remaining objects, and show a success [Toast](/components/toast) informing them that the object has successfully been deleted. If it is possible to undo the deletion, give the user the option to do so, and tell them how long they have to undo the deletion if it is time-bound.
-
-If the delete action fails, keep the modal open and display an error [Toast](/components/toast) that explains what went wrong and how to try again.
-
-For more information, check out our [Notifications and Feedback patterns](/patterns/notifications-and-feedback).
+## After deletion
+
+After the user deletes the object, navigate them to the original page where they can see a list of all remaining objects.
+
+If the delete is successful, show a success [Toast](/components/toast). If the user can undo the deletion, give the user the option to do so in the toast and remove any automatic time out settings. If the undo option is time-bound, tell the user how long they have to undo the deletion.
+
+If the deletion fails, display an error Toast that follows the [error state pattern](/patterns/error-state).
+
+## Do and don't
+
+<DoDont>
+  <Do 
+  title="Do" 
+  body="Use an [Alert Dialog](/components/alert-dialog) for deletion confirmations." 
+  preview={false}
+  />
+  <Dont
+    title="Don't"
+    body="Don’t use a [Modal](/components/modal) for deletion confirmations."
```

Comment:
issue: You probably have to remove these links since links don't work in the Do and Don't components yet 😬

Reply:
done.